rules/do-you-know-when-to-mockup-your-api/rule.md

```diff
@@ -1,27 +1,132 @@
 ---
 type: rule
 archivedreason: 
-title: Comments - Do you know when to mockup your APIs?
+title: Do you know when to mockup your API?
 guid: eee9e703-e6bd-40f8-9f65-8a12a3d92f8d
 uri: do-you-know-when-to-mockup-your-apis
 created: 2021-05-11T22:00:01.0000000Z
 authors:
+- title: William Liebenberg
+  url: https://ssw.com.au/people/william-liebenberg
 - title: Bryden Oliver
   url: https://ssw.com.au/people/bryden-oliver
 related: 
 - do-you-know-how-to-brand-your-api-portal
 - do-you-know-the-best-tools-to-manage-apis
 redirects:
-- comments-do-you-know-when-to-mockup-your-apis
+- do-you-know-when-to-mockup-your-apis
 
 ---
 
-Designing your API up front can be very useful where you have a significant amount of backend development to complete before there is sufficient functionality for front end development to easily commence. 
+Designing your API up front can be very useful where you have a significant amount of backend development to complete before there is sufficient functionality for front end development to easily commence.
+
 <!--endintro-->
-Rather than waiting for the backend to be complete, the APIs can be agreed up front. Tools like **Azure API Management** then allow you to mock up the APIs with sample data, allowing the front end development to commence much sooner.  
- This can result in valuable feedback on the API and backend functionality much earlier in your development lifecycle. The key benefit is being able to present the frontend to stakeholders for feedback really early.  
 
+Rather than waiting for the backend to be complete, the APIs can be agreed on up front. Tools like Azure API Management then allow you to mock up the APIs with sample data, allowing the front end development to commence much sooner.
+
+This can result in valuable feedback on the API and backend functionality much earlier in your development lifecycle. The key benefit is being able to present the frontend to stakeholders for feedback really early.
 
 Watch Adam Cogan and William Liebenberg discuss the benefits.
 
-`youtube: https://youtu.be/nKb6kj_lXXM`
+`youtube: https://youtu.be/rNyOALskc_U`
+
+## Steps to create and test a mocked API endpoint
+
+To complete the following steps, you need to create an instance of Azure API Management in your subscription.
+
+> TIP: If you have Azure Credits, the Developer (NO SLA) edition is the cheapest option to test most of the features offered by API Management.
+
+### Step 1 - Create Blank API
+
+1. Select **Blank API**
+
+![](step1-create-blank-api.png)
+Figure: Create a blank API
```

Comment:
The Figure statement should be put in the alt text
Change from 
```md
![](step1-create-blank-api.png)
Figure: Create a blank API
```
to
```md
![Figure: Create a blank API](step1-create-blank-api.png)
```

see the example rule as a guide https://github.com/SSWConsulting/SSW.Rules.Content/edit/main/rules/rule/rule.md